app/messaging/schemas/funding.js

```diff
@@ -0,0 +1,16 @@
+const Joi = require('joi')
+
+module.exports = Joi.object({
+  name: Joi.string().required(),
+  level: Joi.string().optional().allow(''),
+  area: Joi.string().required(),
+  rate: Joi.string().optional().allow(''),
+  annualValue: Joi.string().required(),
+  quarterlyValue: Joi.string().required(),
+  quarterlyReduction: Joi.string().required(),
+  quarterlyPayment: Joi.string().required(),
+  reductions: Joi.array().items(Joi.object({
+    reason: Joi.string().required(),
+    value: Joi.string().required()
+  })).optional()
+})
```

Comment:
required or optional?

Reply:
The values are required within the array item.  The array itself is optional.

Reply:
sorry, I was alluding to adding `.optional()` to the end of line 16 as done in the other schemas

Reply:
Got you! `funding` and `payments` are array items rather than child properties so don't need `required` or `optional`.  Within `statement` have specified there must be at least 1.

For `scheme` etc I could move required to be referenced in statement instead if you think that's clearer?